configs/dsdl/voc07.py

```diff
@@ -0,0 +1,98 @@
+_base_ = [
+    '../_base_/models/faster-rcnn_r50_fpn.py', '../_base_/default_runtime.py'
+]
+
+# model setting
+model = dict(roi_head=dict(bbox_head=dict(num_classes=20)))
+
+# dataset settings
+dataset_type = 'DSDLDetDataset'
+data_root = '~/.dsdl/datasets/PascalVOC2007-detection'
```

Comment:
Why is this path different from the others?

Reply:
this is just a firrerent root path for dataset, (for users who download dsdl dataset by our command-tools, odl-cli, "~/.dsdl/datasets/" is the default save path )

Reply:
But this is different from other configs like dsdl/coco.py
Furthermore, we should tell users where to find the downloaded dataset in readme